real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyConfigFile.java

```diff
@@ -30,20 +44,267 @@
+			setCOM_PORT_GPS(removeQuotes(configJson.get("COM_PORT_GPS").toString()));
+			setCOM_PORT_RBSM(removeQuotes(configJson.get("COM_PORT_RBSM").toString()));
+			setPORT_VISION(removeQuotes(configJson.get("PORT_VISION").toString()));
+			setWAYPOINT_SOURCE_LOG_FILE(removeQuotes(configJson.get("WAYPOINT_SOURCE_LOG_FILE").toString()));
+			setPLAY_BACK_SOURCE_FILE(removeQuotes(configJson.get("PLAY_BACK_SOURCE_FILE").toString()));
+
+			
+		} catch (UnsupportedEncodingException | FileNotFoundException e) {
+			new RobobuggyLogicNotification("could not load Part of configFile"+e.toString(), RobobuggyMessageLevel.WARNING);
+		}
+	}
+
+	/**
+	 * evaluates to the current value of imuEnabled
+	 * @return
+	 */
+	public static boolean isImuEnabled() {
+		return imuEnabled;
+	}
+
+	/**
+	 * sets imuEnabled to a new value
+	 * @param imuEnabled
+	 */
+	public static void setImuEnabled(boolean imuEnabled) {
+		RobobuggyConfigFile.imuEnabled = imuEnabled;
+	}
+
+	/**
+	 * evaluates to the current value of dataPlayBack
+	 * @return
+	 */
+	public static boolean isDataPlayBack() {
+		return DATA_PLAY_BACK;
+	}
+
+	/**
+	 * evaluates to the current value of comPortImu 
+	 * @return
+	 */
+	public static String getCOM_PORT_IMU() {
+		return comPortImu;
+	}
+
+	/**
+	 * sets a new value to comPortImu
+	 * @param cOM_PORT_IMU
+	 */
+	public static void setCOM_PORT_IMU(String cOM_PORT_IMU) {
+		comPortImu = cOM_PORT_IMU;
+	}
+
+	/**
+	 * evaluates to the current value of configFile
+	 * @return
+	 */
+	public static String getCONFIG_FILE() {
+		return configFile;
+	}
+
+	/**
+	 * sets a new value to configFile
+	 * @param cONFIG_FILE
+	 */
+	public static void setCONFIG_FILE(String cONFIG_FILE) {
+		configFile = cONFIG_FILE;
+	}
+
+	/**
+	 * evaluates to the current value of waypointSourceLogFile
+	 * @return
+	 */
+	public static String getWAYPOINT_SOURCE_LOG_FILE() {
+		return waypointSourceLogFile;
+	}
+
+	/**
+	 * sets a new value to waypointSourceLogFile
+	 * @param wAYPOINT_SOURCE_LOG_FILE
+	 */
+	public static void setWAYPOINT_SOURCE_LOG_FILE(
+			String wAYPOINT_SOURCE_LOG_FILE) {
+		waypointSourceLogFile = wAYPOINT_SOURCE_LOG_FILE;
+	}
+
+	/**
+     * evaluates to the current value of playBackSourceFile
+	 * @return
+	 */
+	public static String getPLAY_BACK_SOURCE_FILE() {
+		return playBackSourceFile;
+	}
+
+	/**
+	 * sets a new value to playBackSourceFile
+	 * @param pLAY_BACK_SOURCE_FILE
+	 */
+	public static void setPLAY_BACK_SOURCE_FILE(String pLAY_BACK_SOURCE_FILE) {
+		playBackSourceFile = pLAY_BACK_SOURCE_FILE;
+	}
+
+	/**
+	 * evaluates to the currentValue of comPortGps
+	 * @return
+	 */
+	public static String getCOM_PORT_GPS() {
+		return comPortGps;
+	}
+
+	/**
+	 * sets a  new value to comPortGps
+	 * @param cOM_PORT_GPS
+	 */
+	public static void setCOM_PORT_GPS(String cOM_PORT_GPS) {
+		comPortGps = cOM_PORT_GPS;
+	}
+
+	/**
+	 * evaluates to the current value of gpsEnabled
+	 * @return
+	 */
+	public static boolean isGps_enabled() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2254452545/issues/source?bid=2938665&fileBranchId=3180106#l233)